phpunit.xml.dist

```diff
@@ -36,4 +36,7 @@
     <logging>
         <junit outputFile="build/report.junit.xml"/>
     </logging>
+    <php>
+        <env name="APP_DEBUG" value="true"/>
```

Comment:
There was an unintended issue with the `responsecache.add_cache_time_header` test code because `APP_DEBUG` was not enabled in all the test code.

However, I'm not sure what differences from before caused this.